webapp/channels/src/components/widgets/menu/menu_items/submenu_item.test.tsx

```diff
@@ -60,8 +74,9 @@ describe('components/widgets/menu/menu_items/submenu_item', () => {
         const action2 = jest.fn();
         const action3 = jest.fn();
 
-        render(
+        render(withIntl(
```

Comment:
Could this use `renderWithContext` instead of `withIntl`? I'm hoping we can get rid of `intl-test-helper` with Enzyme eventually